assets/js/modules/adsense/components/module/ModuleOverviewWidget/index.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * ModuleOverviewWidget component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useState, useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { STORE_NAME } from '../../../datastore/constants';
+import { CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
+import { isZeroReport } from '../../../util';
+import ProgressBar from '../../../../../components/ProgressBar';
+import Header from './Header';
+import Overview from './Overview';
+import Stats from './Stats';
+import Data from 'googlesitekit-data';
+const { useSelect } = Data;
+
+const ModuleOverviewWidget = ( { Widget, WidgetReportZero, WidgetReportError } ) => {
+	const metrics = {
+		EARNINGS: __( 'Earnings', 'google-site-kit' ),
+		PAGE_VIEWS_RPM: __( 'Page RPM', 'google-site-kit' ),
+		IMPRESSIONS: __( 'Impressions', 'google-site-kit' ),
+		PAGE_VIEWS_CTR: __( 'Page CTR', 'google-site-kit' ),
+	};
```

Comment:
Static things like this no longer need to be defined within the component because translations will always be loaded before the JS entry point is initialized. Not a big deal but keeps the component body a little cleaner 🙂

Reply:
Sounds good. Updated!